lib/pengine/utils.c

```diff
@@ -1077,6 +1077,34 @@ unpack_operation(pe_action_t * action, xmlNode * xml_obj, pe_resource_t * contai
         g_hash_table_remove(action->meta, field);
     }
 
+    /*
+     * Timeout order of precedence:
+     *   1. pcmk_monitor_timeout (if rsc has pcmk_ra_cap_fence_params)
+     *   2. explicit op timeout on the primitive
+     *   3. default op timeout
+     *      a. if probe, then min-interval monitor's timeout
+     *      b. else, in XML_CIB_TAG_OPCONFIG
+     *   4. CRM_DEFAULT_OP_TIMEOUT_S
+     *
+     * #1 overrides general rule of <op> XML property having highest
```

Comment:
We can remove this note as gratuitous if you want. My concern is that this precedence table conflicts with L1059:
```
/* Anything set as an <op> XML property has highest precedence.
```

So I thought I'd address that.